src/Pokemon.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Pokedex;
+
+class Pokemon {
+
+    private $id;
+    private $name;
+    private $type_1;
+    private $type_2;
+
+
+    public function __construct()
+    {
+        //capitalises first letter of each string, because it being lowercase looks wroooong
```

Comment:
Consider removing comment

Reply:
changed (but not removed) in ac1a997

I added a comment because someone looking through this later might not realise that the strings aren't capitalised in the db and get rid of it. ucfirst isn't a very descriptive function name